src/server/qgsbufferserverresponse.cpp

```diff
@@ -0,0 +1,161 @@
+/***************************************************************************
+                          qgsfcgiserverresponse.cpp
+
+  Define response wrapper for fcgi response
+  -------------------
+  begin                : 2017-01-03
+  copyright            : (C) 2017 by David Marteau
+  email                : david dot marteau at 3liz dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsbufferserverresponse.h"
+#include "qgslogger.h"
+#include "qgsmessagelog.h"
+
+#include <QDebug>
+
+//
+// QgsBufferServerResponse
+//
+
+QgsBufferServerResponse::QgsBufferServerResponse()
+{
+  mBuffer.open( QIODevice::ReadWrite );
+  mHeadersSent = false;
```

Comment:
Should either be an initializer list, or even better c++11  class member initializers (Define value in header - see http://www.informit.com/articles/article.aspx?p=1852519)